File: 74751_lab1_task5.py
# 74751_lab1_task5.py
import math

r = float(input("Enter body resistance (R): "))
xc = float(input("Enter reactance (Xc): "))
height = float(input("Enter height (in cm): "))

z = math.sqrt(r**2 + xc**2)
tbw = (0.6 * height**2) / z

print(f"Impedance (Z): {z}")
print(f"Total Body Water (TBW): {tbw}")

File: 74751_lab1_task5.cpp
// 74751_lab1_task5.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    double r, xc, height;
    cout << "Enter body resistance (R): ";
    cin >> r;
    cout << "Enter reactance (Xc): ";
    cin >> xc;
    cout << "Enter height (in cm): ";
    cin >> height;
    
    double z = sqrt(r*r + xc*xc);
    double tbw = (0.6 * height * height) / z;
    
    cout << "Impedance (Z): " << z << endl;
    cout << "Total Body Water (TBW): " << tbw << endl;
    
    return 0;
}
